app/components/forms/wizard/basic-details-step.js

```diff
@@ -134,7 +134,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   // TODO: Removing the Event Time Validations due to the weird and buggy behaviour. Will be restored once a perfect solution is found. Please check issue: https://github.com/fossasia/open-event-frontend/issues/3667
   getValidationRules() {
     $.fn.form.settings.rules.checkMaxMinPrice = () => {
-      return $('.ui.form').form('get value', 'min_price') <= $('.ui.form').form('get value', 'max_price');
+      if ($('.ui.form').form('get value', 'min_price') > $('.ui.form').form('get value', 'max_price')) {
+         return false;
+      }
+      else {
+         return true;
```

Comment:
Codacy found an issue: [Expected indentation of 8 spaces but found 9.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5731101)